oaknational/Oak-Web-Application
# Feat/lesq 128/lesson navbar

## Description

- List of changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1783--oak-web-application.netlify.thenational.academy/beta/teachers/programmes/science-secondary-ks3/units/cells-u2b1h4c/lessons/microscopes-lcj273#video
2. Click on left hand nav links
3. You should see the navigation happen

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Acceptance Criteria:

- [x]  Sticky to top of viewport
- [x]  Set column widths
- [x]  Change style on the button of the component you have selected
- [x]  Don’t display the nav menu in mobile (for now)
- [x]  Focus should move to the section you have navigated to rather than staying on the nav bar

## Checklist
